### ISSUE TITLE
ordered-imports doesn't work properly with path separator

### ISSUE THREAD

**Issue by ratoaq2** [palantir/tslint#3662]
### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__:  2.4.2
- __Running TSLint via__: Node.js API and Atom

#### TypeScript code being linted

```ts
import { NgModule } from '@angular/core';
import { NgReduxRouterModule } from '@angular-redux/router';
import { NgRedux, NgReduxModule } from '@angular-redux/store';
```

with `tslint.json` configuration:

```json
    "ordered-imports": [
      true,
      {
        "grouped-imports": true,
        "import-sources-order": "case-insensitive",
        "named-imports-order": "case-insensitive",
        "module-source-path": "full"
      }
    ]
```

#### Actual behavior
`@angular-redux` is happening before `@angular`
```ts
import { NgReduxRouterModule } from '@angular-redux/router';
import { NgRedux, NgReduxModule } from '@angular-redux/store';
import { NgModule } from '@angular/core';
```

#### Expected behavior
`@angular` should come before `@angular-redux`
```ts
import { NgModule } from '@angular/core';
import { NgReduxRouterModule } from '@angular-redux/router';
import { NgRedux, NgReduxModule } from '@angular-redux/store';
```

**Comment by ddagsan** [comment]
Isn't there any solution?

**Comment by JoshuaKGoldberg** [comment]
None so far. There are several rules that sort by names, and it'd be good to have a solid proposal on how to standardize their behavior. Should there be a unified "sort" function used by all rules for consistency? Should rules intentionally differ from each other to have better behavior per rule?

Related: #4063.

**Comment by abierbaum** [comment]
In the meantime you could use #4134 to setup distinct ordered groups.  So it would put "@angular" before "@angular-" packages for example.  It would leave a break between them as groups though so it isn't quite what you need but may be something to consider.

**Comment by adidahiya** [comment]
I'm not convinced that there should be special alphabetization behavior for ordered-imports. I think #4063 is the more important issue to tackle related to this. If you really want `@angular/core` to appear before `@angular-redux/router` (which is not the order produced by `Array.prototype.sort([ '@angular-redux/router', '@angular/core' ])`), you should use the grouping functionality introduced by #4134.
